### ISSUE TITLE
Loss = nan occurred while reproducing the code

### ISSUE THREAD

**Issue by Griffience** [Shujun-He/RibonanzaNet#10]
Hello predecessors, I used L20, fp16 mixed precision, single gpu to run Ribonanza, I did not modify the hyperparameters, why did the problem of loss = nan occur when more than 10 epochs?
你好前辈，我用了L20，fp16混合精度，单gpu跑Ribonanza，我并未对超参数进行修改，为何到了10几个epoch的时候出现了loss = nan的问题呢？
